2022/day06/day06.py

```diff
@@ -1,8 +1,10 @@
 
+
 def is_unique(str: str):
     if len(str) == len(set(str)):
         return True
-        
+
+
 def subroutine(str: str, length: int):
```

Comment:
This function name is rather vague. Try to use more expressive names that will make sense if someone reads them far away from where they're defined. `is_unique` is good. What about something like `find_signal_endpoint`?

Reply:
Also, `str` is a builtin identifier for the string data type. You don't want to define variables with names that are already used by the Python language. This can break Python within your code (now any time you want to use the `str` data type you have no way to refer to it because you'll always be referring to your variable).